v2/examples/example_tls_server_test.go

```diff
@@ -0,0 +1,15 @@
+package examples_test
+
+import (
+	"context"
+
+	"github.com/spiffe/go-spiffe/v2/tlsworkload"
+)
+
+func Example_tLSListener() {
```

Comment:
there is another `Example_tLSListener` maybe we can call it `Example_tLSServer`?

Reply:
I was renaming this file and accidentally saved again afterwards. I'll remove this.